ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/CommandStoreTest.java

```diff
@@ -96,7 +97,7 @@
   private final CommandId commandId =
       new CommandId(CommandId.Type.STREAM, "foo", CommandId.Action.CREATE);
   private final Command command =
-      new Command(statementText, Collections.emptyMap(), Collections.emptyMap());
+      new Command(statementText, Collections.emptyMap(), Collections.emptyMap(), Checksum.none());
```

Comment:
Given the command store will always be storing commands with checksums, is it not worth testing with a checksum and ensuring its stored?

Reply:
Also... just a thought - but maybe the CommandStore itself should throw an exception if we try to enqueue a command without a checksum....